int32 priority
int32 push_target_id
geometry_msgs/Pose2D goal_pose
geometry_msgs/Pose2D start_pose_min_theta
geometry_msgs/Pose2D start_pose_max_theta